src/shared/prerendered-app/Intro/style.css

```
@@ -247,3 +248,13 @@
     display: none;
   }
 }
+
+.sample-button {
+  &:focus {
+    outline: 0;
```

Was this tested with the Windows High Contrast Mode? Using a box shadow instead of an outline usually doesn't work in WHCM, so one needs to provide a [workaround](https://stackoverflow.com/questions/52589391/css-box-shadow-vs-outline/52616313#52616313):

```suggestion
    outline: transparent solid 2px;
```

---

Great idea! Thank you for the suggestion. :)